libindy/tests/utils/constants.rs

```diff
@@ -23,9 +24,12 @@ pub const INVALID_IDENTIFIER: &'static str = "invalid_base58_identifier";
 pub const DEST: &'static str = "FYmoFw55GeQH7SRFa37dkx1d2dZ3zUF8ckg7wmL7ofN4";
 pub const DEST_V1: &'static str = "did:sov:FYmoFw55GeQH7SRFa37dkx1d2dZ3zUF8ckg7wmL7ofN4";
 pub const GVT_SCHEMA_NAME: &'static str = "gvt";
+pub const GVT_SUB_SCHEMA_NAME: &'static str = "gvtsub";
```

Comment:
What is the reason to introduce new schema?

Reply:
To have a credential with partial matching fields and try to cheat on it.